src/overlays/actors/ovl_Obj_Etcetera/z_obj_etcetera.c

```diff
@@ -208,20 +221,21 @@ void ObjEtcetera_DoIntenseOscillation(ObjEtcetera* this, GlobalContext* globalCt
         Actor_SetScale(&this->dyna.actor, 0.01f);
         this->dyna.actor.scale.y = 0.02f;
         this->oscillationTimer = 0;
-        this->intenseOscillationScale = 0.0f;
+        this->bounceOscillationScale = 0.0f;
         return;
     }
-    this->intenseOscillationScale *= 0.8f;
-    this->intenseOscillationScale -= (this->dyna.actor.scale.x - 0.01f) * 0.4f;
-    scaleTemp = dyna->actor.scale.x + this->intenseOscillationScale;
+    this->bounceOscillationScale *= 0.8f;
+    this->bounceOscillationScale -= (this->dyna.actor.scale.x - 0.01f) * 0.4f;
+    scaleTemp = dyna->actor.scale.x + this->bounceOscillationScale;
     Actor_SetScale(&this->dyna.actor, scaleTemp);
     this->dyna.actor.scale.y = 2.0f * scaleTemp;
 }
 
 void ObjEtcetera_Setup(ObjEtcetera* this, GlobalContext* globalCtx) {
     CollisionHeader* colHeader = NULL;
     s32 type;
-    CollisionHeader* collisionHeaders[] = { &D_0400E710, &D_0400E710, &D_040118D8, &D_040118D8 };
+    CollisionHeader* collisionHeaders[] = { &gPinkDekuFlowerCol, &gPinkDekuFlowerCol, &gGoldDekuFlowerCol,
```

Comment:
With the longer names, this would probably be better formatted with a trailing comma.